orchagent/portsorch.cpp

```diff
@@ -2304,6 +2304,24 @@ bool PortsOrch::initializePort(Port &p)
         p.m_oper_status = SAI_PORT_OPER_STATUS_UP;
     }
 
+    /*
+     * If oper_status is not empty, orchagent is doing warm start, restore hostif oper status.
+     */
+    if (!operStatus.empty())
```

Comment:
is this condition good enough? i wonder if there would be some other flags that could check it is a warm start or not

Reply:
okay i get it. thanks